### DISCUSSION TITLE
What is the purpose of importing Mount in fastapi/routing.py and why is it not used in the code?

### DISCUSSION THREAD

**Discussion by AliYmn** [fastapi/fastapi#12993] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from starlette.routing import Mount as Mount  # noqa
```

### Description

In the **fastapi/routing.py** file, there is an import statement on line 73 that reads:

This line **imports the Mount class from starlette.routing** and aliases it as Mount using the as keyword. However, upon further inspection of the fastapi/routing.py file, it appears that the Mount class is not actually used anywhere in the code.
I am curious about the purpose of importing Mount in this file. Is there a specific reason why it is being imported, even though it is not utilized?

Thank you in advance for your insights and explanations!

### Operating System

macOS

### FastAPI Version

0.115.5

### Pydantic Version

2.8.0

### Python Version

3.12.3

**Comment by nb950** [comment, marked as answer]
have you read this https://fastapi.tiangolo.com/advanced/sub-applications/ ? if you follow definitions of app.mount() you get to Mount class
